src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -539,6 +553,39 @@ const formConfig = {
             const requestsRecords = _.get(`disabilities.${index}.view:uploadPrivateRecords`, formData) === 'no';
             return hasRecords && requestsRecords;
           },
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:description': authorizationToDisclose
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
```

Comment:
Can we reuse the `disabilities` definition here?

Reply:
🤔 what's the advantage in doing so? Would that introduce a bunch of schema that we don't actually want in the UI for this page? I'm probably missing something about how forms work...

Reply:
Yeah, I don't think we want to do that. We need the `disabilities` property so we loop through it appropriately as an array page, but we shouldn't need the schema.